internal/server/data/data.go

```diff
@@ -100,6 +100,11 @@ func getDefaultSortFromType(t interface{}) string {
 		return "name ASC"
 	}
 
+	if _, ok := ty.FieldByName("Email"); ok {
```

Comment:
Now that provider users don't have IDs we need to check for a different field before defaulting to sorting by ID. `email` seems like a logical choice after `name`.

Reply:
hmm. does this table even need sort? email might be ineffective as a sort since it should be the same on all providers.

Reply:
It doesn't need it publicly, but we still need it for the list function which always has pagination, so calling it internally (such as in tests) this is required.